components/InformationTile.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="rectangle-info"><p class="tile-content">{{ textContent }}</p></div>
+</template>
+
+<script lang="ts">
+export default {
+  name: 'RectangleInfo',
+  props: {
+    textContent: {
+      type: String,
+      required: true,
+    }
+  },
+};
+</script>
+
+<style>
+.rectangle-info {
+    width: 312px;
+    height: 312px;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(232, 115, 71, 0.9) 100%);
+    border-radius: 30px;
+    margin: 52px;
+   
+    text-align: center;
```

Comment:
Text should be left aligned. Consider adding some padding so that text doesn't touch the edge of the tile.

<img src="https://user-images.githubusercontent.com/67887553/212192167-6b733f57-bc52-4db1-a0b5-68b2964616bc.png" width="250" />